<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="CustomSlider" name="custom_slider" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>212</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="get_value_button">
      <property name="text">
       <string>Get Value</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="set_value_button">
      <property name="text">
       <string>Set Value</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomSlider</class>
   <extends>QWidget</extends>
   <header>customslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
